### ISSUE TITLE
convert ManipulationStation example to pure python

### ISSUE THREAD

**Issue by RussTedrake** [RobotLocomotion/drake#14992]
This was impossible when the `ManipulationStation` was first born, but is possible now.  This would allow people to change the example for their own use cases, as intended, and avoid the temptation of cramming more into the already convoluted API.

I have a version [here](https://github.com/RussTedrake/manipulation/blob/master/manipulation_station.ipynb).  Will work on migrating it soon.

cc @jwnimmer-tri in case it impacts any of your current work.  (I *think* not)
I'm also open to feedback if anybody thinks that keeping the C++ version is important.

**Comment by jwnimmer-tri** [comment]
> cc @jwnimmer-tri in case it impacts any of your current work. (I think not)

That's right.  As long as we have some demo in Drake that generates image and point cloud messages so we can manually test `drake-visualizer`, I'm good.
